Snapper/Manager.py

```diff
@@ -0,0 +1,74 @@
+from Snapper.Snapper import Snapper
+from Snapper.AssociationMatrix import AssociationMatrix
+from Relationship.Variable import Variable
+from Relationship.SpearframeRelationship import SpearframeRelationship
+
+
+class Manager:
+
+    def __init__(self):
+        self.sensors = []
+        self.route_map = {}
+        self.variables = []
+        self.snapper = Snapper(self)
+        self.matrix = AssociationMatrix()
+
+    def add_sensor(self, sensor):
```

Comment:
Ok, I made the changes. I gave the sensor class a set_snapper_callback method instead of the initialization, but I decided to call it in the Snapper level rather than the Manager level of the hierarchy. I chose this because it presented the fewest issues with updating existing test cases and I believe that it makes the most sense because Manager's add_sensor method always has to call Snapper's add_sensor method as well